Mathlib/Tactic/Linter/Style.lean

```diff
@@ -55,7 +56,17 @@ def setOptionLinter : Linter where run := withSetOptionIn fun stx => do
     if (← MonadState.get).messages.hasErrors then
       return
     if let some head := stx.find? is_set_option then
```

Comment:
... and then here you could match directly on the output of `parse_set_option` and extract the `Syntax` node where to log the exception.

Reply:
Indeed, that is much nicer. Changed.